modules/pam_access/pam_access.c

```diff
@@ -605,7 +605,31 @@ netgroup_match (pam_handle_t *pamh, const char *netgroup,
   return retval;
 }
 
-/* user_match - match a username against one token */
+/* user_name_or_uid_match - match a username or user uid against one token */
+static int
+user_name_or_uid_match (pam_handle_t *pamh, const char *tok, const struct login_info *item)
```

Comment:
For consistency this should keep its name or `group_match` below should be ranemed accordingly too.

Reply:
The method `user_match` was not renamed to `user_name_or_uid_match`.  Instead within `user_match`, the call to `string_match` was replaced with `user_name_or_uid_match`.  Since `group_match` and `user_match` already existed I did not want to rename these functions and potentially make breaking changes elsewhere.

The existing function `group_match` could be renamed to `group_name_or_gid_match`, but the new function `user_name_or_uid_match` cannot be named `user_match` due to a function of that name already existing.  Welcome thoughts on how to proceed.

Reply:
Ah, missed that indirect call/split of `user_match`. Would a similar change for `group_match` be reasonable?

Reply:
Like the idea.  It provides a standard approach for both methods.  Pushed the updates, let me know what you think.